tasks/HumanRobotInteractionChallenge.tex

```diff
@@ -0,0 +1,139 @@
+% human detection and recognition, safe navigation and human-robot interaction with unknown people.
+
+\subsection*{Setup}
+\begin{itemize}
+	\item \textbf{Location}: 
+	\begin{itemize}
+	\item The test takes place in the living room.	
+	\item The robot starts inside the \Arena{} at a predefined location near where guests are sitting.
+	\item A table with drinks is prepared near the living room.
+	\item \textbf{Entrance:} The entrance door is open by default. The team leader can request it to be closed to allow the robot to earn additional points by opening it for the guests.
+	\end{itemize}
+	
+	\item \textbf{People:} 
+	\begin{itemize}
+	\item \textbf{Arriving Guests:} Both guests have a name and a favorite drink. An arriving guest will wait by the door, and the robot has to move to meet guests by the door. A guest will not move inside the house until the robot has initiated conversation. It is not allowed to instruct guests to move in front of the robot to recognize the guest and start the conversation. The robot is the one that must approach the guest. Guests have to be guided to the living room where the robot will offer a free place to sit and introduce the guests to each other. Each guest will arrive separately. 
+	\item \textbf{Passive Guests:} A few individuals, including possibly spectators, can be anywhere in the arena — sitting or standing in relevant places, e.g., on the couch, near the seats, or in the area with the drinks.
+	\end{itemize}
+
+\end{itemize}
+
+\subsection*{Procedure}
+Both guests arrive separately. A doorbell will ring to signal the guest has arrived. 
+The doorbell will only ring if the robot is waiting for a new guest at the starting position.
+The robot either opens the door for the guest or meets them at the door.
+It greets the guest and asks for their name and favorite drink.
+The robot escorts the guests to the living room and offers a free seat. 
+Once both guests are seated, the robot introduces them to each other. To introduce the two guests, the robot must (for both guests) look at a guest and state the other arriving guest name and favorite drink.
+The second guest will bring a bag for the host. The robot should ask for the bag to be handed over and move to the host. The host is already waiting and instructs the robot to follow him/her. The host will move to a random place in the house and inform the robot it can drop the bag there.   
+\begin{itemize}
+
+	\item \textbf{Doorbell:} A doorbell sound will be played when a guest is by the door ready to be met. The doorbell sound will not be provided beforehand and may be any traditional doorbell sound. As an alternative, if no doorbell sound is available, the guest can knock on the door. Waiting for the doorbell is optional, robots can bypass it and directly meet the guest at the door. In order for the doorbell sound to be considered detected, the robot must be waiting at the starting position and react instantly to the sound.
```

Comment:
I still think this is kinda awkward regarding setup (Providing sound example, speaker system). Doorbell is one of those things a _deployed_ robot would just get wireless. I think just having the person knock make this much easier during the competition. 


IMHO being able to hear someone shouting at the robot and perceiving environment sound is something we can all imagine our ideal robot to have.  I think we should reintroduce stuff like sound source localization in next years revamped HRI task.
Currently the Robot are mostly tuned for interacting with close persons (and filtering the loud competition background noise) i think 30 Points are not going the influence the design of the robots.

Anyway ill approve this task regardless of choice here.